src/app/booking/OOSPlannerSection.js

```diff
@@ -225,6 +273,51 @@ function useAwaitingPaidStatusBill(billIdInitial = null) {
   const { setBillId } = useBillPolling({ billIdInitial, onSuccess });
 }
 
+function FriendsBookings( {children, weekDates, bookings} ) {
+  const [selectedChild, setSelectedChild] = useState(children[0]);
+  return (
+    <>
+      <TableRow>
+        <StyledTableCell rowSpan={2} sx={{ verticalAlign: "bottom", textAlign: "center" }}>
```

Comment:
Why we need rowSpan here?

Reply:
rowSpan is used to merge two rows. If you remove it, the buttons will be displayed below. verticalAlign for the vertical alignment of elements, in this case we can delete it because it changes nothing